tasks/40_find-friend/find-friend.js

```diff
@@ -0,0 +1,16 @@
+const BLUE = "blue";
+
+export function findFriend(array) {
+  return array.reduce((friends, element, index) => {
+    if (element === "red") {
+      if (
+        (array[index + 1] === BLUE && array[index + 2] === BLUE) ||
+        (array[index - 2] === BLUE && array[index - 1] === BLUE) ||
+        (array[index - 1] === BLUE && array[index + 1] === BLUE)
```

Comment:
Could you condition with comparation to some function?
This code is quite hard to read.

Reply:
I changed it, is that what you meant?